app/assets/stylesheets/mumuki_laboratory/application/modules/_discussion.scss

```diff
@@ -106,11 +98,30 @@ $moderator-badge-color: #dd9900;
     font-size: 18px;
     font-weight: bold;
   }
+  .discussion-title-icons {
+    display: flex;
+    flex-shrink: 0;
+    align-items: center;
+    font-size: calc(#{$font-size-base} * 0.95);
+
+    > * {
+      padding-left: 20px;
+    }
+  }
   .discussion-description {
     overflow: hidden;
     display: block;
     text-overflow: ellipsis;
     white-space: nowrap;
+    margin-bottom: 20px;
+    font-size: calc(#{$font-size-base} * 0.95);
+  }
+  .discussion-initiator {
+    font-size: calc(#{$font-size-base} * 0.85);
```

Comment:
I *really* want to shrink our base font size by 1px. It'll still be clear and easy to read while giving us a little bit more space to work with, especially on mobile screens. Using this variable lets me not-break-everything if I finally change the font size.

Reply:
I am not asking to change this here, but I'd really recommend reinforcing the policy of adding `mu-` prefix to new css classes as opposed to legacy css code which lacked of good semantics, was difficult to customize and allowed name-clashes.

Reply:
Since the whole file is full with classes that lack the prefix, I wrote this issue that we can tackle later. https://github.com/mumuki/mumuki-laboratory/issues/1646